projects/L4T/devices/Switch/packages/moonlight/package.mk

```diff
@@ -0,0 +1,43 @@
+################################################################################
+#      This file is part of OpenELEC - http://www.openelec.tv
+#      Copyright (C) 2009-2012 Stephan Raue (stephan@openelec.tv)
+#
+#  This Program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2, or (at your option)
+#  any later version.
+#
+#  This Program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with OpenELEC.tv; see the file COPYING.  If not, write to
+#  the Free Software Foundation, 51 Franklin Street, Suite 500, Boston, MA 02110, USA.
+#  http://www.gnu.org/copyleft/gpl.html
+################################################################################
+
+PKG_NAME="moonlight"
+PKG_VERSION="20d2d77"
+PKG_ARCH="any"
```

Comment:
if we're planning on moving this to the main packages/libretro/ folder, this ARCH should be set to only 'aarch64'

Reply:
This build is very L4T specific, and was not intended for anything other than the switch/jetson-nano. Putting it with other libretro cores and setting that flag to aarch64 would build and install a broken core on all arm64 boards. Safer to keep this here, and add support for this branch to main tree, but it only supports x86_64 - https://github.com/rock88/moonlight_libretro. Might also need makefile for linux added.